packages/mongodb-memory-server-core/src/MongoMemoryServer.ts

```diff
@@ -54,12 +61,33 @@ export default class MongoMemoryServer {
         console.log(msg);
       }
     };
-    if (!(opts && opts.autoStart === false)) {
+    if (opts?.autoStart === true) {
       this.debug('Autostarting MongoDB instance...');
       this.start();
     }
   }
 
+  /**
+   * Create an Mongo-Memory-Sever Instance that can be awaited
+   * @param opts Mongo-Memory-Sever Options
+   */
+  static async create(opts?: MongoMemoryServerOptsT): Promise<MongoMemoryServer> {
+    // create an instance WITHOUT autoStart so that the user can await it
+    const instance = new MongoMemoryServer({
+      ...opts,
+      autoStart: false,
```

Comment:
Why you decide to disable `autoStart` for this method?

There is no sense to make such method if `autoStart` is `false`.

Maybe we need to switch it to `true` by default?

Reply:
no, it is explicitly set, because a constructor cannot be awaited, and in create `.start` is called and awaited if the original `autoStart` is true

Reply:
Gotcha, tnx!